CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 2.9.9 (2024-07-04)
+
+### All Changes
+
+#### Loki
+
+##### Fixes
+
+* [12925](https://github.com/grafana/loki/pull/12925) **grobinson-grafana** Ingester: Add ingester_chunks_flush_failures_total
+* [13140](https://github.com/grafana/loki/pull/13140) **grobinson-grafana** Ingester: Add backoff to flush op
+
```

Comment:
something is off here
where is the all other changes in between 2.9.4 and 2.9.9?

Reply:
That is a good question, all absent from 2.9.x branch 🤔 

https://github.com/grafana/loki/blob/release-2.9.x/CHANGELOG.md